src/app/users/UsersChefDetails.jsx

```diff
@@ -42,21 +24,22 @@ class UsersChefDetails extends React.Component {
   }
 
   render() {
-    const { currentChef, chefDetails, userDetails } = this.state;
+    const { userDetails } = this.state;
+    const { chef } = this.props.location.state;
     return (
       <div>
-        <h3>{`Chef ${currentChef}`}</h3>
+        <h3>{`Chef ${chef.name}`}</h3>
```

Comment:
Eslint always makes me split things on to different lines, but I haven't been using this format. I should start doing this.